services/PostService.js

```diff
@@ -280,15 +280,29 @@ async function getPostsByQuery(query, clickNo) {
   }
 }
 
-async function getPostCountByUserId(userId) {
+async function getPostCountByUserName(username) {
   let url = "";
   if (typeof window == "undefined") url = serverUrl;
   else url = baseUrl;
   try {
     const { data } = await axios.get(`${url}/${allPostsUrl}/count`, {
-      params: { userId: userId }
+      params: { username: username }
+    });
+    return data;
+  } catch (err) {
+    console.log(err);
```

Comment:
If you return nothing from the catch block you won't be able to handle the error properly in the component.

Reply:
return err.message